### PR TITLE
Create hardware.txt

### PR DESCRIPTION
**Pull request by austin-currington**
Just testing by making placeholder for any hardware docs we want to hold onto

### CODE REVIEW

File: docs/hardware.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test from austin 3.7.22
```
Comment (andrew-fennell) [inline docs/hardware.txt:1]: no

Comment (andrew-fennell) [inline docs/hardware.txt:1]: no

**Review by andrew-fennell — CHANGES_REQUESTED** [review]
idk man

File: docs/hardware.txt (hunk 1 above)
Comment (austin-currington) [reply docs/hardware.txt:1]: yes

**Pull request closed without merging**